draft-ietf-ppm-dap.md

```diff
@@ -400,6 +402,29 @@ to error types, rather than the full URNs. For example, an "error of type
 This document uses the verbs "abort" and "alert with `[some error message]`" to
 describe how protocol participants react to various error conditions.
 
+## HTTPS Sender Authentication {#https-sender-auth}
+
+Some HTTP requests in the PPM protocol require the sender to authenticate its
+request. It does so as described here.
+
+Prior to the start of the protocol, the sender and receiver arrange to share a
+bearer token. The bearer token is an opaque, 32-byte string that MUST be
+generated using a cryptographically secure random number generator.
```

Comment:
```suggestion
Prior to the start of the protocol, the sender and receiver arrange to share a
secret sender-specific API token, which MUST be suitable for representation in
an HTTP header.
```

Reply:
First, the actual guidance is not good:
1. 32 bytes is far longer than needed
2. Binary strings are hard to cut and paste, so people will want to actually have them in ASCII, which this prohibits, and even if it didn't then would then have to re-encode them per the rules below.
3. It forbids structure (e.g., the account ID) which might be convenient.
4. If people have existing systems for managing API tokens, this might conflict.

Second, it's unenforceable and not required for interoperability, so we it's not our job to tell people what to do.

Reply:
Good points.